src/containers/shared/components/Transaction/XChainModifyBridge/test/XChainModifyBridgeSimple.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount, ReactWrapper } from 'enzyme'
+import { Simple } from '../Simple'
+import mockXChainModifyBridge from './mock_data/XChainModifyBridge.json'
+import mockXChainModifyBridgeMinAccountCreateAmount from './mock_data/XChainModifyBridgeMinAccountCreateAmount.json'
+import summarizeTransaction from '../../../../../../rippled/lib/txSummary'
+import i18n from '../../../../../../i18nTestConfig'
+
+function createWrapper(tx: any) {
+  const data = summarizeTransaction(tx, true)
+  return mount(
+    <I18nextProvider i18n={i18n}>
+      <BrowserRouter>
+        <Simple data={data.details} />
+      </BrowserRouter>
+    </I18nextProvider>,
+  )
+}
+
+function expectText(
```

Comment:
This could be super useful in a testing.  Maybe call it expectSimpleRowText.

Reply:
Is there anywhere to put testing utils rn?

Reply:
There is not.  Next week i will create one.  I already have something to put in there